wp-content/themes/chrisdinh-photography/resources/scripts/components/handleGalleryCollectionFilterClick.js

```diff
@@ -0,0 +1,22 @@
+const handleGalleryCollectionFilterClick = () => {
+  const galleryCollections = document.querySelectorAll('button.gallery-collections__collection');
+
+  if (galleryCollections.length < 1) return;
+
+  galleryCollections.forEach(filter => {
+    filter.addEventListener('click', (e) => {
+      e.preventDefault();
+
+      galleryCollections.forEach(filter => {
+        if ( filter.classList.contains('gallery-collections__collection--active') ) {
+          filter.classList.remove('gallery-collections__collection--active');
+          filter.classList.remove('border-b');
+        }
+      });
+
+      filter.classList.add('gallery-collections__collection--active');
```

Comment:
Currently, this loops through all filters to check for the 'gallery-collections__collection--active' class every time a filter is clicked. Could this be optimized to reduce checking all filter buttons to improve scalability?

Reply:
This file may be stale because we're using nanoGallery to handle filtering - i'll remove the file and leave a comment if that's the case.

To your comment - I can run a `filter` method to return the filter element containing `gallery-collections__collection--active` to remove the active class and border. Good suggestion

Reply:
Noted, also figured it was stale due to nanoGallery.